Extensions/fonts/freetype.lisp

```diff
@@ -355,8 +359,8 @@ or NIL if the current transformation is the identity transformation."
                  with rx = 0
                  with ry = 0
                  for current-index in index-list
-                 do (let ((x-pos (+ x rx (glyph-entry-x-offset current-index)))
-                          (y-pos (+ y ry (glyph-entry-y-offset current-index))))
+                 do (let ((x-pos (+ x rx #+nil(glyph-entry-x-offset current-index)))
```

Comment:
use `#+(or)` to comment out form.

Reply:
Fixed in 74453b0977fe7ccba9b1f123d96fd00a665070a1